src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrent.cs

```diff
@@ -227,6 +219,17 @@ private ValidationFailure TestConnection()
                         DetailedDescription = "Sonarr will not attempt to import completed downloads without a category."
                     };
                 }
+
+                // complain if qBittorrent is configured to remove torrents on max ratio
+                var config = _proxy.GetConfig(Settings);
+                if (config.MaxRatioEnabled && config.RemoveOnMaxRatio)
+                {
+                    return new NzbDroneValidationFailure(String.Empty, "QBittorrent is configured to remove torrents when they reach their Share Ratio Limit")
+                    {
+                        IsWarning = true,
```

Comment:
Sorry, comparing to SABnzbd we throw an error if they enable TV sorting, which causes a number of issues with Sonarr, so a think an error makes sense.